File: python/python-read-all-numbers-from-file/example2.py
# https://codevscolor.com/python-read-all-numbers-from-file

try:
    given_file = open("inputs.txt", "r")

    lines = given_file.readlines()

    for line in lines:
        for c in line:
            if c.isdigit() == True:
                print(f"Integer found : {c}")

    given_file.close()
except FileNotFoundError as e:
    print(e)

File: cpp/c-plus-plus-print-odd-numbers/example2.cpp
// https://codevscolor.com/c-plus-plus-print-odd-numbers
#include <iostream>
using namespace std;

int main()
{
    int i = 1;

    while (i <= 100)
    {
        if (i % 2 == 1)
        {
            cout << i << " ";
        }
        i++;
    }

    return 0;
}